language/borrow-graph/src/graph.rs

```diff
@@ -65,6 +65,39 @@ impl<Loc: Copy, Lbl: Clone + Ord> BorrowGraph<Loc, Lbl> {
         (full_borrows, field_borrows)
     }
 
+    pub fn between_edges(&self, parent: RefID, child: RefID) -> Vec<(Loc, Path<Lbl>, bool)> {
+        let mut returned_edges = vec![];
+        let edges = &self.0.get(&parent).unwrap().borrowed_by.0[&child];
+        for edge in edges {
+            returned_edges.push((edge.loc, edge.path.clone(), edge.strong));
+        }
+        returned_edges
+    }
+
+    pub fn out_edges(&self, id: RefID) -> Vec<(Loc, Path<Lbl>, bool, RefID)> {
+        let mut returned_edges = vec![];
+        let borrowed_by = &self.0.get(&id).unwrap().borrowed_by;
+        for (borrower, edges) in &borrowed_by.0 {
+            let borrower = *borrower;
+            let edges: &BTreeSet<BorrowEdge<Loc, Lbl>> = edges;
```

Comment:
nit remove binding (it shouldn't change aything?)

Reply:
I copied the binding from borrowed_by.  Perhaps it should be removed there also.

Reply:
Yeah we should also remove it there if we can

Reply:
Done.